app/javascript/topics/components/TopicsShow__PostShow.re

```diff
@@ -272,29 +272,32 @@ let make =
                        className="leading-normal text-sm "
                        profile=Markdown.QuestionAndAnswer
                      />
-                     {post |> Post.edited
-                        ? <div>
-                            <div
-                              className="text-xs mt-1 inline-block px-2 py-1 rounded bg-gray-100 text-xs text-gray-800 ">
-                              <span> {"Last edited by " |> str} </span>
-                              <span className="font-semibold">
-                                {(
-                                   switch (editor) {
-                                   | Some(user) => user |> User.name
-                                   | None => "Deleted User"
-                                   }
-                                 )
-                                 |> str}
-                              </span>
-                              <span>
-                                {" on "
-                                 ++ Post.updatedAt(post)
-                                    ->DateFns.format("do MMMM, yyyy HH:mm")
-                                 |> str}
-                              </span>
-                            </div>
+                     {switch (Post.editedAt(post)) {
```

Comment:
This whole issue seems to stem from the fact that `Post.edited` is written incorrectly. In fact, I distinctly remember pointing this out in an earlier review. 🤔 

This is what the function says right now:

```reason
let edited = t => {
  t.createdAt != t.updatedAt;
};
```

It's a comparison between `createdAt` and `updatedAt`, when `Post.t` has a perfectly usable `editorId: option(string)`, which is supposed to be set only after a post has actually been _edited_ by someone... unlike `updatedAt` which could change in a few _other_ situations.

However, even if this check had been against the correct marker (`editorId`), it would still need the actual timestamp of when the edit happened... meaning that the changes in `ShowPresenter` to introduce the new `editedAt` field is still probably necessary. 🤔

Reply:
@harigopal: The reason for relying just on `editedAt`, is that `editorId` could be `nil` if the user is deleted. So easy to rely on one field to decide if a post was edited?

Reply:
Ooh. Hmm. Okay. Well, keeping that in mind, now that the new `editedAt` is here, that works best for detecting an edit, I guess.